src/main/scala/dameo/Hash.scala

```diff
@@ -960,110 +960,2110 @@ private object ZobristTables {
-    "01860725034b0fefadd607ff5aaaf995",
-    "f74d369066ec4e96d41ec439c73a3e0f",
-    "1ae9962c6e0d12323ee343c1d9837a9e",
-    "5d66fa465ccfc56097707414bf743321",
-    "e9c13ae1fc36afaae6a8ec453ed67917",
-    "caec4035fb840be403845f0849e183df",
-    "839d28adafad0f8fccd5f9e7ce7e601f",
-    "e4703b6e30422003e2c04ba3484232d8",
-    "1e2fd5b2827d5e435d9b925a3e6af022",
-    "96f1e8d8b94bd960a852b77063b9e148",
-    "90f2075c3f43960c279d2c0c53ecaac6",
-    "c48e0774c4f9134f39f0827a4f811c72",
-    "f17e5f6a2cb000c73b064b8614517b48",
-    "6248409bf55a4925db19bcf000dd394a",
-    "967bd94eb30505cca63dddb617c59634",
-    "e91e89853f9e844fece83ffa46fd173a",
-    "b841038e24193f0801c4d4d39c11557f",
-    "46f3b25cae82a6cca618dd21a5f6b67f",
-    "3e97e042449e3ed58356bdf440563fb0",
-    "868a166af46dcbd2f35ad55d727351a1",
-    "f71be788b3fd1a7aa3a0c83ec173d41c",
-    "cb6d65410533cc37a8b2fe93fc5ddd19",
-    "7e30d70559efaedc984fd612fa3936bd",
-    "32db0f5ca18159ce71c7e142bb029d2a",
-    "97a9116e874228c5759f8a9460c634a6",
-    "85ee68ee3a1752976fa2ca2ca3bdeb90",
-    "076a14170b409e2aea5f0228fc314b97",
-    "bad49d47dc95855bc40c265d97e763c0",
-    "636187d94ded991e201d68d0d89f0e31",
-    "962e50971f09cfab1b769456c77879cf",
-    "8f16c910d6776589a8baaf8b31da09a5",
-    "7e3de4bfbef5566f06a79ac414c9632e",
-    "b262e9f9d61233206e21a47d5b561a1d",
-    "91533947cdaa8bec4263a757e414fe44",
-    "a13b56b45723a3d493c43f67cf55b53f",
-    "9a35cce29ca3ac75a89732f339d35eec",
-    "2716940e1d4f28d75df4ac25c29fbebf",
-    "7447209cfb79306662059230cedcd78f",
-    "5cf91d8ae6402e1a9b0f7261932d2c8e",
-    "4625588d38487ac5fc0f99f00e0cc0e7",
-    "e42ec6191353e3bdcba8a5a02c351aa5",
-    "478e6cc8f6b2dada77a6c01bd0174d69",
-    "1726fc948b994b87c200e5264100e463",
-    "fb9d2e5a66b467413f340a89f525effe",
-    "7f668e401ffe9e6ff748b2be597b2f7a",
-    "ee4d6fe11c46a236f7b2eddf7b8838c2",
-    "006cb70064259959bdec7e9f4317a678",
-    "33535a7c4def1b245e022239fdf20b36",
-    "479e792f8171fc29fdd92fa05b03b629",
-    "656a6e71de97097580cdab95a89927dc",
-    "cada3e48618a1c2b92cb516b8eba4a30",
-    "b37ad7262db9c99eac950f8bce3af2d9",
-    "85ae25402a311d5d9a43060d3aaae01a",
-    "3de4e82d52dbb44cf12b2f6012f9333c",
-    "b1c8499674464c218ae9143ece61584a",
-    "f1c1853cc6827b84676f70930e72993c",
-    "51f97ed3ba004fb025e0f5f014476a1f",
-    "00da9ede878e3e981e33827f042de978",
-    "3cd0fd658e1cdb12dd158e4a7838524d",
-    "ac2940b688a1d0f92b75316dfa1b15e2",
-    "e51acb5b336db0df0283b57f325ea495",
-    "cf7517fbdcb16174146e60f56ab91765",
-    "dfe901aba4a2ced3f50d2497f8b12819",
-    "24bfd4b72c8852eb7600c53f3c60308b",
-    "f085bcd9711883d4b75208a6056dc7e9",
-    "41b71908a3d86274e5f0eb83c20e921b",
-    "6d604cc0a2df1a697529d0a0c95f08ed",
-    "aedf8291e0048c3933c7b70ee04a511c",
-    "09d3c83f59547935727a256dad06cc11",
-    "257d5c7ebc7182421b26058e1ba73008",
-    "56ac1c998f5c2ede09c48fc7e167f3b0",
-    "a25c0b0679937316954f57cdf6076cd4"
+    "433f2611aa7573839b25d1691bb954c0",
```

Comment:
Why is the Hash so big? I realise that it isn't well documented how to calculate the number of hash entries that are needed here, but it should be equal to the max value `actorIndex` can generate:
```
  private def pieceIndex(piece: Piece) =
    piece.role.hashInt * 2 + piece.player.fold(1, 0)
  private def actorIndex(actor: Actor) =
    Pos.all.size * pieceIndex(actor.piece) + actor.pos.hashCode
```
which I think here is:
```
100 * (6*2 + 1) + 99
= 1399
```